README.md

```diff
@@ -40,7 +40,7 @@ or for the cool kids:
 ```shell
 mint n <my-project>
 cd <my-project>
-mint r -cp
+mint r
```

Comment:
I honestly feel like we should just have this info in the `--help` command. What do you think?

Reply:
Maybe both I like it in the README